src/components/Dapp.re

```diff
@@ -177,20 +177,22 @@ module DefaultLook = {
 module DefaultLeftPanel = {
   [@react.component]
   let make = () => {
+    open ReactTranslate;
+    let usedtranslationModeContext = useTranslationModeContext();
+    let translation =
+      useTranslate(. usedtranslationModeContext->translationMode);
     <React.Fragment>
       <h1 className=Styles.heading>
         <span className=Styles.colorBlue> <S> "Always for sale" </S> </span>
         <br />
-        <S> "ethereum based" </S>
+        <S> {translation(. "ethereum") ++ " based"} </S>
```

Comment:
We can switch this to use reason string templates `{j|@$variable1 some tex $variable2 .|j}`